apps/server/src/test-adapter/vc.ts

```diff
@@ -1,17 +1,18 @@
-import { VCAdapterInput, VCDataTypes } from "@repo/utils";
+import { type VCAdapterInput, VCDataTypes } from "@repo/utils";
 import {
   accountsResponse,
   identityResponse,
   transactionsResponse,
+  transcationsByConnectionData,
```

Comment:
This is a typo, but I know this is probably part of the sync so we should probably make a note to fix this on the main repo.